kong/pdk/request.lua

```diff
@@ -820,7 +820,6 @@ local function new(self)
   -- body.age  -- "42"
   function _REQUEST.get_body(mimetype, max_args, max_allowed_file_size)
     check_phase(before_content)
-
```

Comment:
I think it is a unnecessary change, right?

Reply:
reverted